src/components/KeyboardShortcut/KeyboardShortcut.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Text, vars } from "@saleor/macaw-ui-next";
+import { CSSProperties, ReactElement, ReactNode } from "react";
+import { FormattedMessage } from "react-intl";
+
+const isMac = window.navigator.platform?.toLowerCase().includes("mac");
+
+const kbdStyle: CSSProperties = {
+  display: "inline-block",
+  padding: "2px 5px",
+  fontSize: "10px",
+  fontFamily: "inherit",
+  backgroundColor: vars.colors.background.default2,
+  borderRadius: "3px",
+  border: `1px solid ${vars.colors.border.default1}`,
+};
+
+const getModifierKey = (): string => (isMac ? "⌘" : "Ctrl");
+
+interface KeyboardKeyProps {
+  children: ReactNode;
+}
+
+export const KeyboardKey = ({ children }: KeyboardKeyProps): ReactElement => (
+  <kbd style={kbdStyle}>{children}</kbd>
+);
+
+interface KeyboardShortcutHintProps {
+  visible?: boolean;
+}
+
+/**
+ * Displays a keyboard shortcut hint (e.g., "Press ⌘ ↵ to send").
+ * Automatically uses the correct modifier key for the platform.
+ */
+export const KeyboardShortcutHint = ({
```

Comment:
1. I think better name it "KeyboardShortcut" because hint sounds like some tooltip around it
2. It hardcodes `↵`, this should be used from props